create_table.lua

```diff
@@ -193,9 +198,26 @@ for _, tblname in ipairs(tables) do
         end
         local tuple = box.tuple.new(t)
 
-        box.space[tblname:upper()]:insert(tuple)
+        local rc, msg = pcall(box.space[sql_table].insert, box.space[sql_table], tuple)
+        if not rc then
+            print(('database %s failed to insert tuple %s: %s'):
+                format(sql_table, require'json'.encode(tuple), msg))
```

Comment:
These two lines seem rather strange to me, since something is wrong with the indentation, and it seems strange to me to move the method to the next line. It might be worth changing it to something like:
```
            local str = 'database %s failed to insert tuple %s: %s'
            print(str:format(sql_table, require'json'.encode(tuple), msg))
```